app/src/main/java/com/asfoundation/wallet/ui/iab/AsfInAppPurchaseInteractor.java

```diff
@@ -269,6 +270,8 @@ private CurrentPaymentStep map(Transaction transaction, Boolean isBuyReady)
             } else {
               return isBuyReady ? CurrentPaymentStep.READY : CurrentPaymentStep.NO_FUNDS;
             }
+          case myappcoins:
+            return CurrentPaymentStep.PAUSED_LOCAL_PAYMENT;
           default:
           case unknown:
```

Comment:
If we leave "default" as the last condition it will increase readability.
Also, the last 4(COMPLETED, FAILED, CANCELED, INVALID_TRANSACTION) and  cases can be merged leaving only one "return".